### PR TITLE
master to main

### PR DESCRIPTION
**Pull request by ChiefMateStarbuck**

### CODE REVIEW

**Review by scotchneat — CHANGES_REQUESTED** [review]
The included changes look good but I did a search and found several other instances where `master` would need to be changed to `main`.

* commands/apps_test.go [L45](https://github.com/digitalocean/doctl/blob/af5bae4c4a6bdbe0492cc006f76c638bbac63578/commands/apps_test.go#L45) & [L334](https://github.com/digitalocean/doctl/blob/af5bae4c4a6bdbe0492cc006f76c638bbac63578/commands/apps_test.go#L334 (this is a ref to digitalocean/sample-golang which has already changed their default to main))
* scripts/version.sh [L30](https://github.com/digitalocean/doctl/blob/5d5d2fae0eadc58d77f9472375cb1b504a434347/scripts/version.sh#L30)
* CONTRIBUTING.md L255 (docekrfiles/Dockerfile.snap link target)
* README.md
  * L9 ( src link) - I'm not familiar with how travis.ci handles branch changes
  * L206 (Dockerfile link target)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -252,15 +252,15 @@ snap manually.
...
+[dockerfiles/Dockerfile.snap](https://github.com/digitalocean/doctl/blob/main/dockerfiles/Dockerfile.snap).
 The header of the Dockerfile has hints for updating the image, as well as instructions for building
 the image using `make snap_image`. Once you've built the image, the snap_image target will provide
 instructions for next steps.
 
 ### Updating Homebrew
 
 Using the url and sha from the github release, update the
-[homebrew formula](https://github.com/Homebrew/homebrew-core/blob/master/Formula/doctl.rb).
+[homebrew formula](https://github.com/Homebrew/homebrew-core/blob/main/Formula/doctl.rb).
```
Comment (scotchneat) [inline CONTRIBUTING.md:263]: Looks like `homebrew-core` still uses master.

**Review by scotchneat — DISMISSED** [review]
Included one more suggestion. 
Once this is fixed, I think we need either @andrewsomething or @bentranter to review

**Review by bentranter — CHANGES_REQUESTED** [review]
@ChiefMateStarbuck It looks like there are a few duplicated commits in here from other branches. Could you rebase your branch against `master`?

**Comment by ChiefMateStarbuck** [comment]
https://github.com/digitalocean/doctl/pull/942

**Pull request closed without merging**